lib/bots/syncbot.ts

```diff
@@ -31,6 +33,20 @@ import { Logger, LogLevel } from '../utils/logger';
  * A bot that mirrors threads across services.
  */
 export class SyncBot extends ProcBot {
+	public static getConfig(key: string): string {
+		if (process.env[`SYNCBOT_${key}`] !== undefined) {
+			return process.env[`SYNCBOT_${key}`];
+		}
+		if (process.env.CONFIG_TO_LOAD) {
+			const fileContents = fs.readFileSync(`./configs/${process.env.CONFIG_TO_LOAD}.yml`, 'utf8');
```

Comment:
ProcBots already has a configuration framework (`retrieveConfiguration()`/`processConfiguration()`) that all bots inherit that does all of the heavy lifting for you, why is there extra code here which might become stale?